feathr_project/feathr/job_utils.py

```diff
@@ -9,7 +9,7 @@
 
 def get_result_df(client: FeathrClient) -> pd.DataFrame:
     """Download the job result dataset from cloud as a Pandas dataframe."""
-    res_url = client.get_job_result_uri(block=True, timeout_sec=600)
+    res_url = client.get_job_result_uri(block=True, timeout_sec=1200)
```

Comment:
Can we move make timeout_sec a parameter of get_result_df with default 1200, so that callers can also set it?

Reply:
prob can do in next PR.